### PR TITLE
ADR catch up

### PR DESCRIPTION
**Pull request by eeeady**
Added 2 ADRs outlining pros/cons of terraform and building AMIs.

### CODE REVIEW

**Review by jerels — APPROVED** [review]

File: docs/adr/0002-build-amis.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+Most of the running servers in the ECLKC hosting AWS account have been running for a month or two.
+The oldest instance is from 2015.
+The team has kept these servers running and up to date by executing ansible scripts during regular maitenence windows.
+
+## Decision Drivers <!-- optional -->
+
+* Consistency of servers being stood up.
+* Repeatability of server creation.
+* Future scalability of serers and applications we host.
```
Comment (rahearn) [inline docs/adr/0002-build-amis.md:35]:
```suggestion
* Future scalability of servers and applications we host.
```

**Review by rahearn — APPROVED** [review]
I love the approaches being proposed and the justification and documentation of them is top notch.

File: docs/adr/index.md [hunk 2]
```diff
@@ -1,7 +1,5 @@
-
-
 <!-- adrlog -->
 
-
+- [ADR-0001](0001-use-terraform.md) - Use Terraform to Manage AWS Resources
```
[the remark this answers is not in the document]
Comment (eeeady) [reply docs/adr/index.md:3]: I think my precommit hook is broken. I should have this list in the README.md

[the remark this answers is not in the document]
Comment (eeeady) [reply docs/adr/index.md]: Wound up removing the hook. Need to modify it to work on README.md.

**Review by rahearn — APPROVED** [review]

**Pull request merged**